proto/rill/admin/v1/api.proto

```diff
@@ -1839,14 +1839,20 @@ message GetReportMetaRequest {
   string project_id = 1;
   string branch = 2;
   string report = 3;
-  map<string, string> annotations = 4;
-  google.protobuf.Timestamp execution_time = 5;
+  // 4 was previously used for annotations
+  string ownerID = 5;
+  google.protobuf.Timestamp execution_time = 6;
+  repeated string email_recipients = 7;
 }
 
 message GetReportMetaResponse {
-  string open_url = 1;
-  string export_url = 2;
-  string edit_url = 3;
+  message Urls {
```

Comment:
Can we call this type `URLs`? That's the naming convention we use elsewhere with manual naming (though auto-generated protos revert them to `Urls`, which is annoying).